sendgrid/sendgrid.py

```diff
@@ -58,3 +57,14 @@ def api_key(self, value):
     @property
     def impersonate_subuser(self):
         return self._impersonate_subuser
+
+
+class SendGridClient(SendGridAPIClient):

```

Comment:
Here, we actually want the `SendGridClient` to inherit from the `SendGridAPIClient`.  This will help in using the existing features of the latter class as well as abstract some complexity at the same time.